foreign_cc/private/framework.bzl

```diff
@@ -763,7 +766,8 @@ InputFiles = provider(
             "into $EXT_BUILD_DEPS/include."
         ),
         libs = "Library files built by Bazel. Will be copied into $EXT_BUILD_DEPS/lib.",
-        tools_files = (
+        tools_files = "All files (including runfiles needed for the tools",
```

Comment:
I think that we probably also want a tools_manifests entry here and we should merge this with any additional manifests when we call `ctx.actions.run_shell`. I've not fully groked what the manifests input does to an action as its not well documented.